testing/test_junitxml.py

```diff
@@ -678,7 +681,41 @@ def test_pass():
             pass
     ''')
 
-    result = testdir.runpytest(f,
+    result = testdir.runpytest(
+        f,
         '--dist', 'each', '--tx', '2*popen',
         '--junitxml', testdir.tmpdir.join("test.xml"))
-    assert 'INTERNALERROR' not in str(result.stdout)
\ No newline at end of file
+    assert 'INTERNALERROR' not in result.stdout.str()
+
+
+def test_fancy_items_regression(testdir):
+    ' issue 1259'
```

Comment:
Please use triple quotes for docstrings... I know we don't follow pep-8 as much as we would like, but I think this is a consistent rule :)